packages/yourloops/pages/notifications/notification.tsx

```diff
@@ -269,32 +265,33 @@ export const Notification = (props: NotificationProps): JSX.Element => {
     }
   }
 
-  const closeTeamAcceptDialog = (teamId?: string): void => {
+  const closeTeamAcceptDialog = async (teamId?: string): Promise<void> => {
     setAddTeamDialogVisible(false)
     if (notification.target && teamId && notification.target.id === teamId) {
-      acceptInvitation()
+      await acceptInvitation()
     }
   }
```

Comment:
The condition is now useless since you have splitted cancel and accept actions. A suggestion 
const joinTeam = async (): Promise<void> => {
    await acceptInvitation()
    onCloseDialog()
  }